examples/ethernet_networks.py

```diff
@@ -31,7 +31,8 @@
     "credentials": {
         "userName": "<username>",
         "password": "<password>"
-    }
+    },
+    "api_version": 800
```

Comment:
Can we give a placeholder for this also?

Reply:
I think it's better to keep the latest API version as we have an updated resource module and example file to support the latest version of the API.